drivemount/drive-list.c

```diff
@@ -223,177 +223,174 @@ drive_list_remove (GtkContainer *container, GtkWidget *child)
     self = DRIVE_LIST (container);
     button = DRIVE_BUTTON (child);
     if (button->volume)
-	g_hash_table_remove (self->volumes, button->volume);
+        g_hash_table_remove (self->volumes, button->volume);
     else
-	g_hash_table_remove (self->mounts, button->mount);
+        g_hash_table_remove (self->mounts, button->mount);
 
     if (GTK_CONTAINER_CLASS (drive_list_parent_class)->remove)
-	(* GTK_CONTAINER_CLASS (drive_list_parent_class)->remove) (container,
-								   child);
+        (* GTK_CONTAINER_CLASS (drive_list_parent_class)->remove) (container,
+                                                                   child);
```

Comment:
what is the benefit of breaking the line here?

Reply:
same here :)